app/src/main/java/com/ivy/wallet/RootActivity.kt

```diff
@@ -435,16 +435,18 @@ class RootActivity : AppCompatActivity(), RootScreen {
             if (task.isSuccessful) {
                 // We got the ReviewInfo object
                 val reviewInfo = task.result
-                val flow = manager.launchReviewFlow(this, reviewInfo)
-                flow.addOnCompleteListener {
-                    // The flow has finished. The API does not indicate whether the user
-                    // reviewed or not, or even whether the review dialog was shown. Thus, no
-                    // matter the result, we continue our app flow.
-                    if (dismissReviewCard) {
-                        customerJourneyLogic.dismissCard(CustomerJourneyCardsProvider.rateUsCard())
-                    }
+                reviewInfo.let { review ->
```

Comment:
Why this is changed? Please update your branch and don't change this

Reply:
@ILIYANGERMANOV I don't know why I tried to update my branch but still let me check again